DESCRIPTION

```diff
@@ -11,19 +11,18 @@ Description: Provides plotting funtions for reproducing plots in
     a more gov.uk friendly style than the default ggplot2
     settings.
 License: GPL-3
-Depends:  R (>= 2.10),
+Depends: R (>= 2.10),
 	  ggplot2
-Suggests: dplyr,
-          knitr,
-          markdown,
-          readr,
-          tidyr,
-          visualTest
-Imports:  graphics,
-          testthat,
-          png,
-          devtools
-Remotes:  github::mangothecat/visualTest
+Suggests:
+    rmarkdown,
+    testthat,
+    dplyr,
+    readr,
+    knitr,
+    covr,
+    tidyr,
+    vdiffr
```

Comment:
Does this need shifting to remotes for now? I see that https://github.com/r-lib/vdiffr says in the readme

> Important: The CRAN version no longer works properly. Please use the development version which removes the dependency on the system FreeType library.

Reply:
Good point.